src/shared/locales/enUs/translations/labs/index.ts

```diff
@@ -13,7 +13,8 @@ export default {
     },
     requests: {
       label: 'Lab Requests',
-      new: 'New Lab Request',
+      new: 'Request Lab',
```

Comment:
This does not seem to be related to the issue you were working on?

Reply:
This change was also related to my previous issue #2301 and it should've been merged into master. I am not sure why the commits are shown up here :(